pkg/execution/state/redis_state/queue.go

```diff
@@ -556,6 +553,56 @@ func (q QueuePartition) MarshalBinary() ([]byte, error) {
 	return json.Marshal(q)
 }
 
+func (q *queue) RunJobs(ctx context.Context, workspaceID, workflowID uuid.UUID, runID ulid.ULID, limit, offset int64) ([]osqueue.JobResponse, error) {
+	if limit > 10 || limit <= 0 {
+		limit = 10
```

Comment:
could you make this `10` as a `const`?
would be great to have comments on why this limitation as well